### PR TITLE
Add more examples of absolute time

### PR DESCRIPTION
**Pull request by mjdesa**
In response to [87](https://github.com/influxdb/influxdb.com/issues/87), more examples were added.

### CODE REVIEW

**Comment by mjdesa** [comment]
@beckettsean @gunnaraasen @rkuchan thoughts?

File: content/docs/v0.9/query_language/functions.md [hunk 1]
```diff
@@ -66,6 +76,8 @@ DISTINCT() returns an array of unique values for the given field.
 SELECT DISTINCT(field_key) FROM measurement 
 
 SELECT DISTINCT(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT DISTINCT(field_key) FROM measurement WHERE 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: missing `time >`

Comment (mjdesa) [inline content/docs/v0.9/query_language/functions.md]: whoops good chatch

File: content/docs/v0.9/query_language/data_exploration.md [hunk 2]
```diff
@@ -22,19 +22,19 @@ For Example:
 
 To select from a measurement `(client)`:
 ```sql
-SELECT * FROM "(client)"
+SELECT * FROM "(client)" WHERE time > 1434059627s
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/data_exploration.md]: these are basic examples and shouldn't be expanded in this way, I think. Let's teach one thing at a time

File: content/docs/v0.9/query_language/data_exploration.md [hunk 3]
```diff
@@ -22,19 +22,19 @@ For Example:
...
 ```
 
 To select for a `cpu` measurement with a tag key of `cpu_1` and tag value of `1`:
 ```sql
 SELECT * FROM cpu
-WHERE cpu_1 = '1'
+WHERE cpu_1 = '1' AND time > 1434059627s
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/data_exploration.md]: these are basic examples and shouldn't be expanded in this way, I think. Let's teach one thing at a time

File: content/docs/v0.9/query_language/data_exploration.md [hunk 4]
```diff
@@ -22,19 +22,19 @@ For Example:
...
 ```
 
 To select for a `cpu` measurement with a tag key of `cpu-1` and tag value of `1`:
 ```sql
 SELECT * FROM cpu
-WHERE "cpu-1" = '1'
+WHERE "cpu-1" = '1' AND time > 1434059627s
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/data_exploration.md]: these are basic examples and shouldn't be expanded in this way, I think. Let's teach one thing at a time

File: content/docs/v0.9/query_language/functions.md [hunk 5]
```diff
@@ -16,6 +16,8 @@ COUNT() takes a single field key as the only argument. It returns the number of
 SELECT COUNT(field_key) FROM measurement
 
 SELECT COUNT(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT COUNT(field_key) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 6]
```diff
@@ -26,6 +28,8 @@ MIN() returns the lowest value from the specified field over a given interval. T
 SELECT MIN(field_key) FROM measurement
 
 SELECT MIN(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT MIN(field_key) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 7]
```diff
@@ -36,6 +40,8 @@ MAX() returns the highest value from the specified field over a given interval.
 SELECT MAX(field_key) FROM measurement 
 
 SELECT MAX(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT MAX(field_key) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 8]
```diff
@@ -46,6 +52,8 @@ MEAN() returns the arithmetic mean (average) of the specified field over a given
 SELECT MEAN(field_key) FROM measurement
 
 SELECT MEAN(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT MEAN(field_key) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 9]
```diff
@@ -56,6 +64,8 @@ MEDIAN() returns the middle value from a sorted set of values for the specified
 SELECT MEDIAN(field_key) FROM measurement
 
 SELECT MEDIAN(field_key) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT MEDIAN(field_key) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md (hunk 1 above)
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 10]
```diff
@@ -74,6 +86,8 @@ You can nest distinct in count to get the counts of unique values over windows o
 SELECT COUNT(DISTINCT(field_key)) from measurement
 
 SELECT COUNT(DISTINCT(field_key)) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT COUNT(DISTINCT(field_key)) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md [hunk 11]
```diff
@@ -84,6 +98,8 @@ PERCENTILE() returns the Nth percentile value of a sorted set of values for the
 SELECT PERCENTILE(field_key, N) FROM measurement
 
 SELECT PERCENTILE(field_key, N) FROM measurement WHERE time > now() - 1d GROUP BY time(10m)
+
+SELECT PERCENTILE(field_key, N) FROM measurement WHERE time > 1434059627s GROUP BY time(10m)
```
Comment (beckettsean) [inline content/docs/v0.9/query_language/functions.md]: let's use a different `GROUP BY` clause for these examples, like `GROUP BY tag`

File: content/docs/v0.9/query_language/functions.md (hunk 5 above)
Comment (mjdesa) [inline content/docs/v0.9/query_language/functions.md]: In both examples or just the absolute time ones?

**Pull request merged**